File: Practice/21/python/21.py
def BMI(weight: float, height: float) -> float:
    return weight / (height * height / 10000)
def print_BMI(BMI: float) -> float:
    if (BMI < 18.5):
        print("Underweight")
    elif ((BMI >= 18.5) and (BMI < 25)):
        print("Normal")
    elif ((BMI >= 25) and (BMI < 30)):
        print("Overweight")
    else:
        print("Obesity") 
ves, rost = map(float,input().split())
print_BMI(BMI(ves, rost))

File: Practice/21/c++/21/21/21.cpp
#include <iostream>
using namespace std;

double BMI(double weight, double height)
{
	double digit=(weight / (height * height / 10000));
	return digit;
}

void printBMI(double digit)
{
	if (digit < 18.5)
	{
		cout << "Underweight";
	}
	else if ((digit >= 18.5) && (digit < 25))
	{
		cout << "Normal";
	}
	else if ((digit >= 25) && (digit < 30))
	{
		cout << "Overweight";
	}
	else
		cout << "Obesity";
}

int main()
{
	double ves, rost;
	cin >>ves >> rost;
	printBMI(BMI(ves, rost));
}